src/components/overblikk.tsx

```diff
@@ -57,40 +72,42 @@ const Overblikk = () => {
     }, [oppfolgingsstatus?.veilederId]);
 
     // Midlertidig data i overblikk, må endres etterhvert som vi får bedre innsikt i hva som trengs i oversikten
-    const veilederNavn: StringOrNothing = veileder?.navn;
-    const telefon: StringOrNothing = person?.telefon?.[0]?.telefonNr; // Henter kun ut 1 tlfnr, flere kan være registrert
+    const veilederData: VeilederData | null | undefined = veileder;
+    const telefon: StringOrNothing = person?.telefon?.find((entry) => entry.prioritet === '1')?.telefonNr;
     const antallBarn: StringOrNothing = person?.barn?.length.toString();
-    const oppfolgingsenhet: OppfolgingEnhet | undefined = oppfolgingsstatus?.oppfolgingsenhet;
-    const talespraak: StringOrNothing = tolk?.talespraak;
+    const taletolk: OrNothing<TilrettelagtKommunikasjonData> = tolk;
     const sivilstatus: StringOrNothing = person?.sivilstandliste?.[0]?.sivilstand;
-    const brukersMaal: StringOrNothing = oppfolgingsstatus?.hovedmaalkode;
-    const ytelserAktivitetsFase: StringOrNothing = ytelser?.vedtaksliste?.[0]?.aktivitetsfase; // Henter kun ut 1 ytelse, flere kan være registrert
-    const geografiskEnhet: OrNothing<Enhet> = person?.geografiskEnhet;
+    const brukersMaal: OrNothing<Hovedmal | ArenaHovedmalKode> = oppfolgingsstatus?.hovedmaalkode;
+    const ytelserVedtakstype: StringOrNothing | undefined = ytelser?.vedtaksliste
```

Comment:
`StringOrNothing`-typen inkluderer allerede `undefined` så dere trenger ikke spesifisere det her:

```suggestion
    const ytelserVedtakstype: StringOrNothing = ytelser?.vedtaksliste
```